src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenAddPaymentTest.java

```diff
@@ -51,12 +55,13 @@
 
     @Test
     void shouldAddConfigurationToConfigBuilder() {
-        final Set<State> stateSet = Set.of(State.class.getEnumConstants());
-        final ConfigBuilderImpl<CaseData, State, UserRole> configBuilder = new ConfigBuilderImpl<>(CaseData.class, stateSet);
+        final ConfigBuilderImpl<CaseData, State, UserRole> configBuilder = createCaseDataConfigBuilder();
 
         citizenAddPayment.configure(configBuilder);
 
-        assertThat(configBuilder.getEvents().get(0).getId(), is(CITIZEN_ADD_PAYMENT));
+        Assertions.assertThat(getEventsFrom(configBuilder).values())
```

Comment:
can static import it :)

Reply:
It is probably because there is already a static import of assertThat, I will convert them to AssertJ instead